modules/nlu-testing/src/shared/typings.ts

```diff
@@ -27,3 +28,14 @@ export interface TestResult {
   details: TestResultDetails[]
   nlu: IO.EventUnderstanding
 }
+
+export interface VisData {
+  gt: string
+  pred: string
```

Comment:
same here: prediction ?

Reply:
Yes changed as well for expeted and prediction